src/InterNations/Sniffs/Architecture/ControllerMethodsConventionSniff.php

```diff
@@ -0,0 +1,74 @@
+<?php
+namespace InterNations\Sniffs\Architecture;
+
+use PHP_CodeSniffer_File as CodeSnifferFile;
+use PHP_CodeSniffer_Sniff as CodeSnifferSniff;
+
+class ControllerMethodsConventionSniff implements CodeSnifferSniff
+{
+    private static $apiVerbs = ['index', 'get', 'post', 'put', 'patch', 'delete'];
+
+    private static $webVerbs = ['new', 'edit'];
```

Comment:
These should not exist. If we're gonna be anal about controller method names, we better be anal all the way towards an established standard (rest in this case) and not our own smorgasbord.